src/logic/safe/hooks/useLoadSafe.tsx

```diff
@@ -13,6 +13,12 @@ export const useLoadSafe = (safeAddress?: string): boolean => {
   const dispatch = useDispatch<Dispatch>()
   const [isSafeLoaded, setIsSafeLoaded] = useState(false)
 
+  useEffect(() => {
+    if (isSafeLoaded) {
+      setIsSafeLoaded(false)
+    }
+  }, [safeAddress])
```

Comment:
I'm not sure if the `isSafeLoaded` check is required. What would be the situation when the address changes but safe is already loaded, so we don't need to set it to `false`?

Reply:
Also, there's an eslint warning

Reply:
I've update this accordingly. Thanks!